components/tabs/tabs.js

```diff
@@ -1001,6 +1003,21 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 		return this.updateComplete;
 	}
 
+	#checkTabPanelMatch() {
+		// debounce so only runs once when tabs/panels slots changing
+		if (this._checkTabPanelMatchRequested) return;
+
+		this._checkTabPanelMatchRequested = true;
```

Comment:
Notice we'd using `_` here for this new private variable. What are your thoughts on `#` - I think that's what I'd normally do now, and we've started to use it for methods here. Do we eventually want to make a sweep through this code and switch, maybe once the clean-up is done? What would be the best timing to do that?

Reply:
I'd generally been using `_` to keep the new `...DefaultSlotBehavior` functions together with the old in the diff, and then was planning a sweeping change later to update to `#`. But yeah I think for cases like this it would make sense just to use `#` now. I'll update that.